docs/api.md

```diff
@@ -945,14 +950,14 @@ An example of dumping frame tree:
 const puppeteer = require('puppeteer');
 
 puppeteer.launch().then(async browser => {
-  let page = await browser.newPage();
+  const page = await browser.newPage();
   await page.goto('https://www.google.com/chrome/browser/canary.html');
   dumpFrameTree(page.mainFrame(), '');
   browser.close();
 
   function dumpFrameTree(frame, indent) {
     console.log(indent + frame.url());
-    for (let child of frame.childFrames())
+    for (const child of frame.childFrames())
```

Comment:
let's keep `let` inside the for - that would be consistent with how we right puppeteer code.